teku/src/main/java/tech/pegasys/teku/cli/subcommand/RemoteSpecLoader.java

```diff
@@ -38,16 +39,16 @@ static Spec getSpecWithRetry(final List<URI> beaconEndpoints) {
 
   static Spec getSpec(final List<URI> beaconEndpoints) {
     if (beaconEndpoints.size() > 1) {
-      return getSpecWithFailovers(createApiClients(beaconEndpoints));
+      return getSpecWithFailovers(createTypeDefClients(beaconEndpoints));
     }
-    return getSpec(createApiClient(beaconEndpoints.get(0)));
+    return getSpec(createTypeDefClient(beaconEndpoints.get(0)));
```

Comment:
this seems like a logic trap we can unwind, calling the single will just create another list, it's a little messy.